src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -156,16 +159,12 @@ public final boolean perform(final AbstractBuild<?, ?> build, final Launcher lau
         String commentAction = "none";
         if (runHarbormaster) {
             logger.info("harbormaster", "Sending build result to Harbormaster with PHID '" + phid + "', success: " + harbormasterSuccess);
-            try {
-                JSONObject result = diffClient.sendHarbormasterMessage(phid, harbormasterSuccess);
-                if (result.containsKey("error_info") && !(result.get("error_info") instanceof JSONNull)) {
-                    logger.info("harbormaster",
-                            String.format("Error from Harbormaster: %s", result.getString("error_info")));
-                    return false;
-                }
-            } catch (ConduitAPIException e) {
-                logger.info(CONDUIT_TAG, "unable to post to harbormaster");
-                return true;
```

Comment:
Slight change in semantics. This used to consider the build "successful" in jenkins even if harbormaster wasn't working. I think the new semantics are better.